applications/GeoMechanicsApplication/tests/cpp_tests/test_newmark_quasistatic_U_Pw_scheme.cpp

```diff
@@ -55,21 +54,15 @@ class NewmarkQuasistaticUPwSchemeTester {
         p_node->AddDof(WATER_PRESSURE);
         result.GetProcessInfo()[DELTA_TIME] = 4.0;
 
-        p_node->FastGetSolutionStepValue(DISPLACEMENT, 1) =
-            Kratos::array_1d<double, 3>{7.0, 8.0, 9.0};
-        p_node->FastGetSolutionStepValue(VELOCITY, 1) =
-            Kratos::array_1d<double, 3>{0.0, 0.0, 0.0};
-        p_node->FastGetSolutionStepValue(ACCELERATION, 1) =
-            Kratos::array_1d<double, 3>{0.0, 0.0, 0.0};
+        p_node->FastGetSolutionStepValue(DISPLACEMENT, 1) = Kratos::array_1d<double, 3>{7.0, 8.0, 9.0};
+        p_node->FastGetSolutionStepValue(VELOCITY, 1) = Kratos::array_1d<double, 3>{0.0, 0.0, 0.0};
```

Comment:
lack of automagic = alignment.

Reply:
As above pointed. I can change it only manually right now.